corgi/collectors/pnc.py

```diff
@@ -2,14 +2,71 @@
 from collections.abc import Mapping
 from typing import Any
 
+from corgi.core.models import Component
+
 logger = logging.getLogger(__name__)
 
 
-def parse_pnc_sbom(data: Mapping[str, Any]) -> int:
-    """Parse an SBOM from PNC and create components
-    Temporarily returns an int for testing purposes, to be
-    completed in CORGI-488"""
-    if "components" not in data:
-        raise ValueError("SBOM is missing component data")
+class SbomerSbom:
+    """Collector to parse SBOMs from PNC/Sbomer. See CORGI-488.
+    Future development should generalize this where applicable
+    to other middleware."""
+
+    def __init__(self, data: Mapping[str, Any]):
+        # Product information
+        root_meta = {
+            param["name"]: param["value"] for param in data["metadata"]["component"]["properties"]
+        }
+        self.product = root_meta["errata-tool-product-name"]
+        self.product_version = root_meta["errata-tool-product-version"]
+        self.product_variant = root_meta["errata-tool-product-variant"]
```

Comment:
There are streams defined in prod defs, but the values provided by sbomer are ET values. Since the components are really only ever related to products through builds via ProductComponentRelations, these values might not even be necessary.